src/main/java/Duke.java

```diff
@@ -1,10 +1,44 @@
+import java.util.Scanner;
```

Comment:
Adding new lines between methods, class definitions, and import statement could help improve readability